web-server/plugins/slycat-cca/js/components/CCAWizardUtils.tsx

```diff
@@ -199,6 +221,78 @@ export const useHandleWizardSetup = (
   }, [statePid, stateMid, dispatch, pid, marking]);
 };
 
+export const selectTableFile = () => {
```

Comment:
this needs to be rename to `useSelectTableFile` or react will freak out. Proper hook naming has the word `use` in front